charts/sptribs-frontend/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: '1.0'
 description: A Helm chart for sptribs-frontend App
 name: sptribs-frontend
 home: https://github.com/hmcts/sptribs-frontend
-version: 0.0.35
+version: 0.0.36
```

Comment:
Think this was bumped when you changed changed the case api url. Make sure to revert this back before merge.